templates/hero.hbs

```diff
@@ -0,0 +1,17 @@
+<section class="hero container">
+   <div class="hero__wrapper d-lg-inline-flex justify-content-left">
+      <h1 class="hero__title col-lg-9">{{{title}}}</h1>
+      <div class="hero__imageWrapper">
+         <img src="{{assetSrc cover}}" class="hero__image"/>
+         <button class="hero__button"><img src="/images/icon-play.svg" alt="play"></button>
+      </div>
+   </div>
+   <div class="hero__scroll justify-content-center d-lg-none">
+      <p class="p-big">SCROLL</p> 
+      <img src="/images/icon-scroll.svg" alt="">
+   </div>
+   <div class="hero__description row justify-content-end">
+      <p class="p-big col-8">{{{description}}}</p>
+   </div>
+   
+</section>
```

Comment:
Please add a wrapper around your elements so that in desktop the background color takes all the spaces around the component. Like this
```suggestion
<section class="hero">
  <div class="container">
    <div class="hero__wrapper d-lg-inline-flex justify-content-left">
      <h1 class="hero__title col-lg-9">{{{title}}}</h1>
      <div class="hero__imageWrapper">
         <img src="{{assetSrc cover}}" class="hero__image"/>
         <button class="hero__button"><img src="/images/icon-play.svg" alt="play"></button>
      </div>
     </div>
     <div class="hero__scroll justify-content-center d-lg-none">
       <p class="p-big">SCROLL</p> 
       <img src="/images/icon-scroll.svg" alt="">
     </div>
     <div class="hero__description row justify-content-end">
       <p class="p-big col-8">{{{description}}}</p>
     </div>
  </div>
</section>
```

Reply:
Fixed